### ISSUE TITLE
Filter before CAR?

### ISSUE THREAD

**Issue by weallen** [cortex-lab/neuropixels#22]
I was debating with Marius about this: is it better to high-pass or band-pass filter the raw data before CARing it, or should the filtering just be done during spike sorting? I had heard from someone that because the filters on the probe aren't great, it's good to remove any low-frequency components of the AP band before CARing, but maybe this isn't correct?

**Comment by nsteinme** [comment]
One important note (Will I'm sure you know this but for others who may not): phase3a probes have an idiosyncratic offset on every channel so that by default the AP band data does not have mean=0. So it's important to subtract that first. 

I doubt the residual low frequency components are big enough to matter much... but it probably doesn't hurt to high-pass filter first. However, I don't - my code for this is [here](https://github.com/cortex-lab/spikes/blob/master/preprocessing/applyCARtoDat.m).

**Comment by weallen** [comment]
Yeah, I always remove the constant offset.  Good to know, thanks!
